modules/ROOT/pages/metrics-archive-api.adoc

```diff
@@ -0,0 +1,33 @@
+= Anypoint Monitoring Archive API
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Anypoint Monitoring Archive API enables you to download metric data for APIs. The data that this API archives appears in your xref:monitoring::api-analytics-dashboard.adoc#enhanced-api-metrics[enhanced API metrics dashboards]. However, the data contained in the archive is asynchronous.
+
+You must have a Titanium subscription to use this API.
+
+== File Structure and Downloads
+
+The Anypoint Monitoring Archive API uses a directory hierarchy. At the lowest level, you can access and download files containing archived data from your enhanced API metrics dashboards.
```

Comment:
We can also point out that the user should be familiar with enhanced API Metrics, the supported Mule runtime versions, and have used advanced custom dashboard features before proceeding with Archive API. By using those, the user will then have the right understanding of the content provided from Archive API as you can see the same content through advanced custom dashboard.